hw/ip/uart/data/uart.prj.hjson

```diff
@@ -20,9 +20,11 @@
       {
         version:            "1.1",
         life_stage:         "L1",
-        design_stage:       "D2S",
+        design_stage:       "D3",
         verification_stage: "V2",
         dif_stage:          "S2",
+        commit_id:          "558942bb7869d9a5d8abd4bd0eb46dab820d3564"
```

Comment:
this would be optional, since we don't require it when moving forward through the dev stages (we only need to record it when we bounce back and create a new version).